examples/chat.rs

```diff
@@ -234,10 +239,21 @@ impl SignedMessage {
     }
 }
 
+/// A chat message.
 #[derive(Debug, Serialize, Deserialize)]
 enum Message {
-    AboutMe { name: String },
-    Message { text: String },
+    AboutMe {
+        name: String,
```

Comment:
This message probably also needs a nonce, since when a new peer joins, you'd want to send yet another `AboutMe` message with your current username, but that'd currently fail sending.

```suggestion
        name: String,
        nonce: [u8; 16],
```